packages/cubejs-server-core/core/index.js

```diff
@@ -162,7 +163,9 @@ class CubejsServerCore {
       this.logger
     );
     apiGateway.initApp(app);
-    this.initDevEnv(app);
+    if (this.options.devServer) {
```

Comment:
There're already these devServer checks in place. I believe we should just replace all `process.env.NODE_ENV !== 'production'` occurrences with `this.options.devServer`.

Reply:
I agree, please see https://github.com/statsbotco/cube.js/pull/29/files#diff-49fd186d45485663b0e641a0214512e2R102 (Line 102)